pyproject.toml

```diff
@@ -16,7 +16,7 @@ classifiers = [
     "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
 ]
-readme = "README.rst"
+readme = "README.md"
```

Comment:
this is left-over from back when pypi didn't support markdown. It looks like they changed that at some point; so long as it shows up fine markdown is cool by me.

Reply:
It works just fine now!
For example: [`glocaltokens`](https://github.com/leikoilja/glocaltokens), rendered the same on [PyPI](https://pypi.org/project/glocaltokens/).